Cant find the module expo-text-extractor

Hey!

I just downloaded the package, and i cant make it work.


This error appears everytime i install the package.

`Cannot find module 'expo-text-extractor' or its corresponding type declarations.ts(2307)
`

I cant find a solution, help please.

--- a/.github/workflows/main.yml
+++ b/.github/workflows/main.yml
@@ -36,6 +36,7 @@ jobs:
           node-version: 20
           cache: 'npm'
       - run: npm ci
+      - run: npm run build
       - env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
           NPM_TOKEN: ${{ secrets.NPM_TOKEN }}

--- a/package.json
+++ b/package.json
@@ -28,6 +28,8 @@
     "open:android": "open -a \"Android Studio\" example/android"
   },
   "devDependencies": {
+    "@auto-it/conventional-commits": "^11.3.0",
+    "@auto-it/npm": "^11.3.0",
     "@types/react": "~18.3.12",
     "auto": "^11.3.0",
     "expo": "~52.0.0",

